python-checks/src/main/java/org/sonar/python/checks/FlaskHardCodedSecret.java

```diff
@@ -162,16 +160,16 @@ private static boolean isSensitiveProperty(Expression expression) {
         .map(ExpressionList::expressions)
         .filter(list -> list.size() == 1)
         .map(list -> list.get(0))
-        .map(FlaskHardCodedSecretCheck::getAssignedValue)
+        .map(FlaskHardCodedSecret::getAssignedValue)
         .flatMap(TreeUtils.toOptionalInstanceOfMapper(StringLiteral.class))
         .map(StringLiteral::trimmedQuotesValue)
-        .filter(SECRET_KEY_KEYWORD::equals)
+        .filter(key -> this.getSecretKeyKeyword().equals(key))
```

Comment:
replace with:
```java
.filter(this.getSecretKeyKeyword()::equals)
```

Reply:
Fixed in 76ae07f32865c40b90d2654034fcbbf935f06188.